scripts/provision-elasticsearch.sh

```diff
@@ -54,7 +55,7 @@ provision_elasticsearch_ubuntu14 () {
     # install Elasticsearch server:
     sudo DEBIAN_FRONTEND=noninteractive apt-get -y install \
          elasticsearch \
-         openjdk-7-jre
+         $openjdk_version
```

Comment:
This would be incomprehensible in the generated documentation... Can you please amend to have explicit version detection?

```
# detect Elasticsearch version
apt-cache search | grep | and| stuff
```

(or checkout `default-jre`?)